apps/condo/bin/notification/send-receipt-added-notifications.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Sends billing receipt added notifications
+ *
+ * Usage:
+ *      yarn workspace @app/condo node bin/send-receipt-added-notifications-to-all-contexts [FORCE_SEND] [<MAX_NOTIFICATIONS_PER_PROPERTY>] [FORCE_UNLOCK] [<RESEND_FROM_DATETIME>]
```

Comment:
👍

Reply:
But also good to specify what force unlock means

Reply:
done